packages/atomic/src/components/result-template-components-v1/atomic-result-multi-text/atomic-result-multi-text.tsx

```diff
@@ -0,0 +1,162 @@
+
+/**
+ * The `atomic-result-multi-text` component renders the values of a multi-value string result field.
+ * @part result-multi-text-separator - The separator between two field values.
+ * @part result-multi-text-value - A field value.
+ * @slot result-multi-text-value-* - Allows the personalization of a particular field value based on its indexed name.
+ */
+@Component({
+  tag: 'atomic-result-multi-text',
+  styleUrl: 'atomic-result-multi-text.pcss',
+  shadow: true,
+})
+export class AtomicResultMultiText implements InitializableComponent {
+  public breadcrumbManager!: BreadcrumbManager;
+
+  @InitializeBindings() public bindings!: Bindings;
+  @ResultContext() private result!: Result;
+
+  @State() public error!: Error;
+
+  /**
+   * The result field which the component should use.
+   * This will look in the Result object first, and then in the Result.raw object for the fields.
+   * It is important to include the necessary field in the ResultList component.
+   */
+  @Prop() public field!: string;
+
+  /**
+   * If present, will attempt to split the field into multiple values using wherever this character is encountered.
+   */
+  @Prop() public delimitingCharacter?: string;
+
+  /**
+   * The maximum number of field values to display.
+   * If there are more values than the maximum that was set here, the last value displayed will be "# more...".
+   */
+  @Prop() public maximumValues = 3;
+
+  public initialize() {
+    this.breadcrumbManager = buildBreadcrumbManager(this.bindings.engine);
+  }
+
+  private get resultValues() {
+    const value = ResultTemplatesHelpers.getResultProperty(
+      this.result,
+      this.field
+    );
+
+    if (Array.isArray(value)) {
+      return value.map((v) => `${v}`);
+    }
+
+    if (typeof value !== 'string' || value.trim() === '') {
+      return [];
+    }
+
+    return this.delimitingCharacter
+      ? value.split(this.delimitingCharacter)
+      : [value];
+  }
+
+  private get facetSelectedValues() {
+    return this.breadcrumbManager.state.facetBreadcrumbs
+      .filter((facet) => facet.field === this.field)
+      .reduce(
+        (values, facet) => [
+          ...values,
+          ...facet.values.map(({value}) => value.value),
+        ],
+        [] as string[]
+      );
+  }
+
+  private getSortedValues() {
+    const allValues = this.resultValues;
+    const allValuesSet = new Set(this.resultValues);
+    const firstValues = this.facetSelectedValues.filter((value) =>
+      allValuesSet.has(value)
+    );
+    return Array.from(
+      allValues.reduce((set, value) => set.add(value), new Set(firstValues))
+    );
+  }
+
+  private renderValue(value: string) {
+    const label = getFieldValueCaption(this.field, value, this.bindings.i18n);
+    const kebabValue = titleToKebab(value);
+    return (
+      <li key={value} part="result-multi-text-value" class={listItemClasses}>
+        <slot name={`result-multi-text-value-${kebabValue}`}>{label}</slot>
```

Comment:
Why is a slot used here? I did not understand why in the PR description:

```
<atomic-result-multi-text field="language">
  <span slot="result-multi-text-value-japanese">🇯🇵 Japanese</span>
</atomic-result-multi-text>
```

a span for japanese is added?

Reply:
The slot allows the customization of specific values.

For instance, in the design, we see this being used to add icons before values:
![image](https://user-images.githubusercontent.com/54454747/129600160-b9e35150-9fea-45f3-91f4-e2afd8adddd5.png)

Reply:
I see, thanks for the clarification